packages/sanity/src/structure/panes/document/inspectors/validation/ValidationInspector.tsx

```diff
@@ -82,14 +82,14 @@ export function ValidationInspector(props: DocumentInspectorProps) {
   )
 }
 
-function ValidationCard(props: {
+export function ValidationCard(props: {
   marker: ValidationMarker
-  onOpen: (path: Path) => void
+  onOpen?: (path: Path) => void
   schemaType: ObjectSchemaType
   value: Partial<SanityDocument> | null
 }) {
   const {marker, onOpen, schemaType, value} = props
-  const handleOpen = useCallback(() => onOpen(marker.path), [marker, onOpen])
+  const handleOpen = useCallback(() => onOpen?.(marker.path), [marker, onOpen])
```

Comment:
Validation card is reused in the table view to render the validation info but it's not receiving any "onOpen" function, given there is no option yet to redirect the user straight to the field.

<img width="837" alt="Screenshot 2024-06-10 at 09 34 44" src="https://github.com/sanity-io/sanity/assets/46196328/54f493ee-586c-4d2e-bf90-751c44e92404">